independent-projects/resteasy-reactive/server/processor/src/main/java/org/jboss/resteasy/reactive/server/processor/ServerEndpointIndexer.java

```diff
@@ -676,6 +678,14 @@ private ParameterConverterSupplier extractConverter(String elementType, IndexVie
                 || elementType.equals(InputStream.class.getName())) {
             // this is handled by MultipartFormParamExtractor
             return null;
+        } else {
```

Comment:
I think something like this would be a better condition:

```java
        } else if (SUPPORT_TEMPORAL_PARAMS.contains(DotName.createSimple(elementType))) {
```

This is what is done in `handleOptionalParam` already.

Reply:
Now should be ok. Thanks